src/telemetry.c

```diff
@@ -330,7 +330,7 @@ void TELEMETRY_SetType(int type)
 //#define DEBUG_TELEMALARM
 void TELEMETRY_Alarm()
 {
-    if (! PROTOCOL_GetTelemetryState())
+    if (PROTOCOL_GetTelemetryState() == PROTO_TELEM_UNSUPPORTED)
```

Comment:
I don't think this is right   shouldn't it be == PROTO_TELEM_OFF

Reply:
Shouldn't it be `if (!(PROTOCOL_GetTelemetryState() == PROTO_TELEM_ON))`?

Reply:
Correct.  Only have telemetry values to alarm on when state is ON.

Reply:
You are right. I am still wondering how can I better test this without real hardware?

Reply:
@hexfet I‘m sorry, it was late for me ;)

Reply:
No worries, everybody does it :)